cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/CashContract.kt

```diff
@@ -18,14 +18,25 @@ class CashContract : Contract {
 
         when (command.value) {
             is Commands.Create -> requireThat {
-                // everyone can print/crate cash... because of covi... crisis
-                val cashState = tx.inputsOfType<CashState>().single()
-                "Cash value must be positive" using (cashState.balance > 0)
+                "There should be no input." using tx.inputsOfType<CashState>().isEmpty()
+                // in this exercise I allow anyone to create cash
+                val cashState = tx.outputsOfType<CashState>().single()
+                "Cash value must be positive" using (cashState.value > 0)
+                "Expect one signature" using (command.signers.size == 1)
+                "Creator must be a signer" using (command.signers.single() == cashState.owner.owningKey)
```

Comment:
Can you separate the cash issuer (the central bank) from the cash owner (you and me)?